portfolio/src/main/webapp/script.js

```diff
@@ -84,4 +89,17 @@ function deleteComment(comment) {
   const params = new URLSearchParams();
   params.append('id', comment.id);
   fetch('/delete-data', {method: 'POST', body: params});
+}
+
+/** Make a GET request to /blobstore-upload-url */
```

Comment:
Nit: URL in comment doesn't match the URL in the function.

Reply:
Fixed!